index.html

```diff
@@ -9,16 +9,135 @@
 </head>
 <body>
     <header>
-        <div id="logo">ABDOU</div>
+        <div id="logo"><a href="#home">ABDOU SHOP</a></div>
         <div id="open-menu"><button id="open-menu-button" type="submit"><img src="img/header/open-menu.png" alt="open menu"></button></div>
-        <div class="search-input"><input type="search" name="search" id="search" placeholder="Search for items and brands"><button type="submit"><img src="img/header/search-icon.png" alt="search button"></button></div>
         <nav class="navbar">
             <div id="close-menu"><button id="close-menu-button" type="submit"><img src="img/header/close-menu.png" alt="close menu"></button></div>
             <ul class="navbar-links">
-                <li><a href="#home">HOME</a></li>
-                <li><a href="#about">ABOUT</a></li>
+                <li><a class="navbar-link" id="first-link" href="#home">HOME</a></li>
+                <li><a class="navbar-link" id="second-link" href="#services">SERVICES</a></li>
+                <li><a class="navbar-link" href="#about">ABOUT</a></li>
             </ul>
         </nav>
     </header>
+
+    <div class="hero" id="home">
+        <h1>DO YOU WANNA LOOK BEAUTIFUL ?</h1>
```

Comment:
- [ ] I think the right color here should be #f43d28. You can use color-picker google extension to get the colors from the design reference.

![image](https://user-images.githubusercontent.com/64894927/133410561-0c02478c-a7b5-40a4-ad27-d443013ccae5.png)